strawberry_django/fields/types.py

```diff
@@ -38,6 +38,11 @@
     IntegerChoicesField = None
     TextChoicesField = None
 
+try:
+    from django.db.models import GeneratedField  # type: ignore
```

Comment:
Worth adding a comment here that `GeneratedField` is only available at Django 5.0+ so that we remember to remove it when dropping support for 4.2 in the future?

Another suggestion would be to change this to:

```python
if django.VERSION >= (5, 0):
    from django.db.models import GeneratedField
else:
    GeneratedField = None
```

This is self-documented, and is the kind of string that I would search for when searching for code that can be dropped/simplified after a version bump